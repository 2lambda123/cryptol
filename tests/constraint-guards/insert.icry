:l insert.cry
:check insert_correct
