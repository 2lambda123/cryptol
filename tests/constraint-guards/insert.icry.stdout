Loading module Cryptol
Loading module Cryptol
Loading module Main
[warning] at insert.cry:3:1--3:7:
  Could not prove that the constraint guards used in defining Main::insert were exhaustive.
Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
